documentation/tutorials/buildlinux.rst

```diff
@@ -0,0 +1,87 @@
+============================================================
+  How to build the Linux kernel for an x86_64 architecture
+============================================================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+.. highlight:: console
+
+Get a Linux repository
+----------------------
+After successfully :ref:`installing kw<install-and-remove-kw>`, it's time to
+build the Linux kernel using it. Let's start by cloning the Linux repository
+from Torvalds into our current working directory::
+
+  git clone https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+
+Then, change the current directory to the Linux repository::
+
+  cd linux
+
+.. note::
+    From now on, all commands will assume you're in the kernel folder.
+
+Get .config
+-----------
+To build the kernel, a `.config` file is necessary. This file contains
+configurations about the kernel compilation process.
+To retrieve this file from your computer, run either::
+
+  cp /boot/config-`uname -r` .config
+
+Or::
+
+  zcat /proc/config.gz > .config
+
+.. note::
+    Get the .config file from the same machine you plan on using the customized
```

Comment:
maybe change machine to linux distribution?

Reply:
yep! that makes things more clear